utils/src/lib.rs

```diff
@@ -53,7 +53,12 @@ pub fn encode_input(param: &ParamType, value: &str) -> Result<Token> {
     Ok(match param {
         // TODO: Do the rest of the types
         ParamType::Address => Address::from_str(value)?.into_token(),
-        ParamType::Bytes => Bytes::from(value.from_hex::<Vec<u8>>()?).into_token(),
+        ParamType::Bytes => {
+            match &value[0..2] {
+                "0x" => Bytes::from(value[2..].from_hex::<Vec<u8>>()?).into_token(),
+                _ => Bytes::from(value.from_hex::<Vec<u8>>()?).into_token(),
+            }
```

Comment:
it's unlikely but this will panic if `len(value) < 2`, 
```suggestion
           value.trim_start_matches("0x").from_hex::<Vec<u8>>()?).into_token()
```

Reply:
ah nice thank you!